### PR TITLE
Acorn+Readme Refactor

### PR DESCRIPTION
**Pull request by mbroecheler**

### CODE REVIEW

File: finance-credit-card-chatbot/analytics-agent/creditcard.google-gemini.config.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "prompt": "You are a helpful customer service representative for a credit card company who helps answer customer questions about their past transactions and spending history. Today's date is January 18th, 2024. You provide precise answers and use functions to look up information. You DO NOT provide general answers and only give an answer after you retrieved all the data you need via functions. Only invoke one function at a time and wait for the results before invoking another function.When you have worked out your answer, you answer the user question in one of two ways: 1) in markdown syntax using tables where appropriate to show data or 2) by calling the data_visualization function to display the data in a suitable fashion. Whenever you are returning multiple data points, you should use option 2) and call the data_visualization function. You should only call the data_visualization function when you already have all the data to display.",
+  "apis": {
+    "type": "graphql",
+    "url": "${env:API_SERVER}"
+  },
+  "plot-function": "oneD",
+  "model" : {
+    "provider" : "vertex",
+    "model": "GEMINI_1_5_FLASH",
```
Comment (mbroecheler) [inline finance-credit-card-chatbot/analytics-agent/creditcard.google-gemini.config.json:10]: @bilbo7833 this looks like an enum value to me. Can you update the Gemini module to use the actual model name like we do for the other models?

File: iot-sensor-metrics/agent/sensors.google-gemini.config.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "apis": {
+    "type": "graphql",
+    "url": "${env:API_SERVER}"
+  },
+  "auth-field": "customerid",
+  "auth-is-integer": true,
+  "plot-function": "oneD",
+  "model" : {
+    "provider" : "vertex",
+    "model": "GEMINI_1_5_FLASH",
```
Comment (mbroecheler) [inline iot-sensor-metrics/agent/sensors.google-gemini.config.json:12]: @bilbo7833 Same here

**Comment by bilbo7833** [comment]
@mbroecheler Sorry, I updated all config files now.

File: finance-credit-card-chatbot/analytics-agent/creditcard.google-gemini.config.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "prompt": "You are a helpful customer service representative for a credit card company who helps answer customer questions about their past transactions and spending history. Today's date is January 18th, 2024. You provide precise answers and use functions to look up information. You DO NOT provide general answers and only give an answer after you retrieved all the data you need via functions. Only invoke one function at a time and wait for the results before invoking another function.When you have worked out your answer, you answer the user question in one of two ways: 1) in markdown syntax using tables where appropriate to show data or 2) by calling the data_visualization function to display the data in a suitable fashion. Whenever you are returning multiple data points, you should use option 2) and call the data_visualization function. You should only call the data_visualization function when you already have all the data to display.",
+  "apis": {
+    "type": "graphql",
+    "url": "http://host.docker.internal:8888/graphql"
```
Comment (bilbo7833) [inline finance-credit-card-chatbot/analytics-agent/creditcard.google-gemini.config.json:5]: @mbroecheler This hostname is not found when I run the backend with the usual commands

**Review by henneberger — APPROVED** [review]

**Pull request merged**